rosdep/base.yaml

```diff
@@ -4104,6 +4099,18 @@ postgresql-client:
   fedora: [postgresql-client]
   gentoo: [dev-db/postgresql]
   ubuntu: [postgresql-client]
+postgresql-postgis:
```

Comment:
this new key seem very similar to `postgresql-9.x-postgis` above.
Though bionic is the first version using 10.X instead of 9.X, so this new key is version agnostic :+1:

@tfoote how do you feel about this ? should we add the EOL rules from https://github.com/ros/rosdistro/blob/8bb80c8ce3bcbb570acf3f9d77e3358c486d64cc/rosdep/base.yaml#L4098 here ? To have one key that works for all distributions even outdated ones ?
Or are you ok merging this as is ?

@cberner can you please apply the fix of the missing packages (contrib and server) for debian to the existing `postgresql-9.x-postgis` key as well? Or confirm that the current rules are equivalent in debian and ubuntu ? (https://github.com/ros/rosdistro/pull/10830)

Reply:
done